docs/pages/reference/cli.mdx

```diff
@@ -2406,7 +2406,8 @@ tctl version
 
 ## Resource filtering
 
-Both `tsh` and `tctl` allow you to filter `nodes`, `apps`, `db`, and `kube` resources using the `--search` and `--query` flags.
+Both `tsh` and `tctl` allow you to filter `nodes`, `apps`, `db`, `pod`, and
```

Comment:
`tctl` does not allow you to search by `pod` since Pod resources do not exist in Teleport cache.
`tsh` does not support it yet but it will soon.